engine/dlib/src/dlib/hash.cpp

```diff
@@ -666,24 +666,31 @@ DM_DLLEXPORT void dmHashReverseErase64(uint64_t hash)
 DM_DLLEXPORT const char* dmHashReverseSafe64(uint64_t hash)
 {
     const char* s = (const char*)dmHashReverse64(hash, 0);
-    if (s == 0)
-    {
-        char tmp[64];
-        dmSnPrintf(tmp, sizeof(tmp), "<unknown:%llu>", (unsigned long long)hash);
-        return tmp;
-    }
-    return s;
+    return s != 0 ? s : "<unknown>";
```

Comment:
In practice, the "s" lifespan is valid until the game (and the reverse table) is shut down.